mobile/mobileConfig.js

```diff
@@ -0,0 +1,17 @@
+const mobileConfig = {
+  icons: {
+    defaultColor: "#FFEDE6",
+    lgIcon: {
+      size: 30,
+      strokeWidth: 2,
+      color: "#FFEDE6"
+    },
+    xlIcon: {
+      size: 33,
+      strokeWidth: 2,
+      color: "#FFEDE6"
+    }
```

Comment:
Same issue as with @BarnabePILLIAUDIN: please refrain from using different files as source of truth: better keep those values where they belong to: a dedicated file for the design, à la tailwind.config.js